boards/arm/nucleo_f767zi/nucleo_f767zi.dts

```diff
@@ -112,3 +113,59 @@
 &rng {
 	status = "okay";
 };
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/*
+		 * The two sectors 0-1 (32+32 kbytes) are reserved for
+		 * the bootloader.
+		 */
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x0 0x00010000>;
+			read-only;
+		};
+
+		/*
+		 * Sectors 2-3 (32+32 kbytes) unallocated.
+		 */
+
+		/*
+		 * The flash starting at offset 0x20000 and ending at
+		 * offset 0x3ffff is reserved for use by the application.
+		 * This represents sector 4 (128 k)
+		 */
+		storage_partition: partition@20000 {
+			label = "storage";
+			reg = <0x00020000 0x00020000>;
+		};
+
```

Comment:
IIRC storage requires minimum of 2 sectors for normal operations.
@Laczen is that still valid ?

Reply:
@nvlsianpu ^^

Reply:
@nandojve, @erwango, for storage it is a bad idea to use only one sector as this does not allow you to retain any data if the sector needs to be erased. Persistent storage always requires at least 2 sectors. When nvs is used it will also not support sectors larger then 32kB. IMO it would be best to use the 2 leftover 32 kB sectors for storage.

Reply:
@Laczen has responded excellent here,